### DISCUSSION TITLE
Does RAG document connector support pdf?

### DISCUSSION THREAD

**Discussion by anastasiakonovalova1989** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#94] [Q&A] [answered]
Good afternoon,

I'm trying to figure out if it is possible to use pdf files within RAG document connector. I saw in documentation and some articles that it should work. But whenever I'm using "pdf" in file type it returns an error:
![pdf](https://github.com/user-attachments/assets/df3d8a8c-a38b-4ef2-a2df-6c4f1dc448bd)
![error](https://github.com/user-attachments/assets/0a9a0754-0a3c-44d8-93cf-10c47ce93164)

If I use File type: text instead, the pdf file is not recognized by LLM model, I guess because for text type we only can use txt, csv etc.

Thank you in advance!

**Comment by amirkhan-ak-sf** [comment]
@anastasiakonovalova1989 please use type `any`. it should work.

**Reply by anastasiakonovalova1989** [reply, marked as answer]
@amirkhan-ak-sf Thank you! it seems to be working well with any.
